domains/faiyaz.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "FaiyazMujawar",
+        "email": "mujawar.faiyaz.aziz@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://me-faiyazmujawars-projects.vercel.app/"
```

We do not support Vercel, also CNAME is incorrect
```suggestion
        "CNAME": "me-faiyazmujawars-projects.vercel.app"
```